PowerEditor/src/WinControls/ProjectPanel/ProjectPanel.cpp

```diff
@@ -44,8 +44,8 @@
 #define INDEX_LEAF           5
 #define INDEX_LEAF_INVALID   6
 
-#define GET_X_LPARAM(lp)     LOWORD(lp)
-#define GET_Y_LPARAM(lp)     HIWORD(lp)
+#define GET_X_LPARAM(lp) static_cast<short>(LOWORD(lp))

```

Comment:
Aren't those already defined in Windows headers?

Reply:
Yes, you're right. I've included the header now. Thanks!